docs/cookbook/legacyDateFromDateTime.mjs

```diff
@@ -0,0 +1,27 @@
+import assert from "assert";
+
+/**
+ * Converts a Temporal.DateTime instance into a Date.
+ * This is the inverse of dateTimeFromLegacyDate example
+ *
+ * @param {Temporal.DateTime} dateTime This is a Date instance
+ * @returns {Date} LegacyDate instance
```

Comment:
Small nitpick, but if we go with legacy as the preferred term, I'd say "legacy Date" here instead of "LegacyDate". LegacyDate makes it sound like there's a class named that.